source/chapters/advanced_topics.rst

```diff
@@ -233,3 +233,32 @@ using sends for effects. This gives Mixxx access to the extensive collection of
 Make sure the correct multichannel audio interface has been selected in Jack
 (Jack settings visible bottom left). Note that Mixxx possibly labels its Jack
 ports as :guilabel:`Portaudio`.
+
+.. _advanced-migrate-settings:
+
+Migrate your Mixxx library and settings to a new computer
+=========================================================
+
+If you move to a new computer, or to another operating system on the same computer, you can take your existing Mixxx setup with you. This includes your track library, settings, controller mappings and broadcast profiles.
```

Comment:
Shouldn't we mention here, that the beatgrid offset depend on the soundsource. And that the library might be unusable on a computer with an other OS or Linux distribution?

Reply:
> the beatgrid offset depend on the soundsource

True. I'll check if shifting the cues is dosumented somewhere, and if we can link there.

> unusable on a computer with an other OS or Linux distribution

Why 'unusable'?

Reply:
I meant, that if there is an offset in the beatgrid, the beatgrid must be adjusted for each track.

Reply:
Ah, okay. I'll add a note.

Reply:
I rebased it on #597 (which can be merged quickly) in order to have a reference to the beatgrid controls.